Assets/AssetDatabase.csv

```diff
@@ -7,6 +7,19 @@ id, hash, doesExist, path
 11157760038865534979,9100331679172514553,1,./Assets/Meshes/Sphere2.obj
 11814002129831317412,4888425721009333584,1,./Assets/Meshes/Sphere3.obj
 5885694396001899252,2022751725491440666,1,./Assets/Meshes/Sphere4.obj
-5642008164619175156,5946622663772039769,1,./Assets/Texture/BlueSky.png
-10398316861534336334,15213441317390681113,1,./Assets/Texture/ColdNight.png
+5642008164619175156,5061962178892228107,1,./Assets/Texture/BlueSky.png
+10398316861534336334,5061962178892228107,1,./Assets/Texture/ColdNight.png
 16923617995496829252,9841322819738978230,1,./Assets/Meshes/Quad.obj
+8311092884496302402,5061962178892228107,0,./Assets/Texture/Bottom.png
+3409684700574447267,11606561679558660453,0,./Assets/Texture/Up.png
+4469883716313123428,9408646944479156035,0,./Assets/Texture/Back.png
+4902418189210660769,6775437063696746205,0,./Assets/Texture/Front.png
+2751923634483836088,5061962178892228107,0,./Assets/Texture/Left.png
+2036820931153573773,5061962178892228107,0,./Assets/Texture/Right.png
+621632243211435008,7972749528915198812,0,./Assets/Texture/Back1.png
```

Comment:
Have you deleted these parts? Because I didn't see them in the texture folder. 
Please delete these lines if you delete the file from texture folder